tests/navigation.rs

```diff
@@ -147,3 +147,33 @@ fn can_navigate_using_breadcrumbs(
 
     Ok(())
 }
+
+#[rstest]
+#[case(server(&["--default-sorting-method", "date", "--default-sorting-order", "desc"]))]
```

Comment:
Can you add a second case where you order the other way and demonstrate that this works?